### PR TITLE
Adds some more syndicate equipment.

### PR DESCRIPTION
**Pull request by WJohn**
Fancy syndicate combat attire with TONS OF TACTICAL POCKETS (designer clothing, these are not actual pockets).

These are not possible to get by normal means, and are purely cosmetic.

Also tweaks a bit of flavor text and equipment for upcoming battlecruiser fun.

### CODE REVIEW

File: code/game/objects/structures/ghost_role_spawners.dm [hunk 1]
```diff
@@ -395,31 +395,49 @@
 	H.faction |= "syndicate"
 
 /obj/effect/mob_spawn/human/syndicate/battlecruiser
-	name = "Syndicate Battlecruiser Ensign"
-	flavour_text = "<font size=3>You are a syndicate ensign aboard a state of the art weapons platform -- the SBC Starfury. <b>Follow your captain's orders, maintain the ship, and protect both with your life.</b>"
+	name = "Syndicate Battlecruiser Ship Operative"
+	flavour_text = "<font size=3>You are a crewmember aboard the syndicate flaship: the SBC Starfury. <span class='danger'><b>Your job is to follow your captain's orders, maintain the ship, and keep the engine running.</b></span> If you are not familiar with how the supermatter engine functions: <b>do not attempt to start it.</b><br><br><span class='danger'><b>The armory is not a candy store, and your role is not to assault the station directly, leave that work to the assault operatives.</b></span></font>"
```
Comment (ChangelingRain) [inline code/game/objects/structures/ghost_role_spawners.dm]: flaship

**Comment by PKPenguin321** [comment]
they look like they're crouching

**Comment by Durkel** [comment]
Now all you need is a fedora and a flame t-shirt to go will all those pockets.

**Comment by ShadowDeath6** [comment]
Bend those knees

**Comment by CosmicScientist** [comment]
Big chest? High hips? That top is weird.

**Comment by WJohn** [comment]
The one on the left is a girl so it looks a bit off, but the chest/legs are at the same height as any typical pants/shirts are.

**Comment by KingPhilipIII** [comment]
If you're doing these sprites, you should really give nukies something like this instead of making the chest rig the same thing as the explorers webbing.

**Comment by WJohn** [comment]
I'm not even the one who did the chest rig recolor, that was kor, then either kevinz or steelpoint who made a grey version.

**Comment by Tacolizard** [comment]
t o p h e a v y o r b o t t o m h e a v y

**Comment by CosmicScientist** [comment]
>one on the left is a girl
>beard
The autogenerator is not conforming to my views!

**Review by Cyberboss — CHANGES_REQUESTED** [review]
DME

**Comment by WJohn** [comment]
IT FINALLY WORKED.

**Pull request merged**